index.html

```diff
@@ -55,6 +54,228 @@ <h2 class="green">I’m a Full Stack developer from Microverse</h2>
     </div>
  
   </section>
+
+
+  <button class="arrow">
+    <img src="images/buttons/Disabled.png" alt="">
+  </button>
+
+  <section class="recent-work">
+    <header>
+      <h2 class="green">
+        My recent work
+      </h2>
+      <img src="images/texts/Indicator.png" alt="">
+    </header>
+
+    <div class="projects">
+
+      <!-- Project 1 -->
+
+      <div class="project">
+        <div class="project-image">
+          <img src="" alt="" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Mobile version skeleton
+          </h3>
+        </div>
+        <ul class="project-built-with">
+          <li><a href="">Ruby on Rails</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">Javascript</a></li>
+          <li><a href="">HTML</a></li>
+        </ul>
+
+        <div class="project-buttons">
+          <a href="">See project</a>
+        </div>
+        
+      </div>
+
+      <!-- Project 2 -->
+
+      <div class="project">
+        <div class="project-image">
+          <img id="img-placeholder" src="images/Img Plaholder.png" alt="" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Mobile version
+          </h3>
+        </div>
+        <ul class="project-built-with">
+          <li><a href="">Ruby on Rails</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">Javascript</a></li>
+          <li><a href="">HTML</a></li>
+        </ul>
+
+        <div class="project-buttons">
+          <a href="">See project</a>
+        </div>
+        
+      </div>
+
+      <!-- Project 3 -->
+
+      <div class="project">
+        <div class="project-image">
+          <img src="" alt="" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Contact form
+          </h3>
+        </div>
+        <ul class="project-built-with">
+          <li><a href="">Ruby on Rails</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">Javascript</a></li>
+          <li><a href="">HTML</a></li>
+        </ul>
+
+        <div class="project-buttons">
+          <a href="">See project</a>
+        </div>
+        
+      </div>
+
+      <!-- Project 4 -->
+
+      <div class="project">
+        <div class="project-image">
+          <img src="" alt="" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Desktop version
+          </h3>
+        </div>
+        <ul class="project-built-with">
+          <li><a href="">Ruby on Rails</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">Javascript</a></li>
+          <li><a href="">HTML</a></li>
+        </ul>
```

Comment:
- Just as the last reviewer mentioned, please ensure that this section is styled precisely as in the Figma template. Please fix this 🙏

Your project:
- Please add a border-radius and ensure they are styled as in the design template 👍 👍 
<img width="273" alt="tetweweg" src="https://user-images.githubusercontent.com/73607512/227745477-048b96fb-b78c-43f6-abd9-cfa25a7e40d4.PNG">

Reply:
Done @mckent05 @mahbubul14